services/dummy-test-service/dbmethods.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+type user struct {
+	UserID   int
+	Username string
+	Nickname string
+	City     []city
+}
+
+type city struct {
+	CityID    int
+	Name      string
+	Buildings []building
+}
+
+type building struct {
+	BuildingID int
+	Name       string
+	Level      int
+	Available  bool
+}
+
+type dbClient interface {
+	getDB() map[string][]user
```

Comment:
Maybe this method is more like: `getUsers` ? 
We still have to define the DB struct. I'd actuall advise agaisnt having all users in the "map", i.e., each user would be one document.

Reply:
So actuall `getDB` should not exit